examples/ads.amp.html

```diff
@@ -1165,6 +1174,12 @@ <h2 id="yahoojp">YahooJP YDN</h2>
       data-yadsid="79712_113431">
   </amp-ad>
 
+  <h2 id="yandex">Yandex</h2>
```

Comment:
got 
![image](https://cloud.githubusercontent.com/assets/5419904/26020036/06bffe5a-372f-11e7-960c-6a464d6b6a9c.png)
Please test the example with our local server.